android/src/main/java/io/kommunicate/app/RNKommunicateChatModule.java

```diff
@@ -227,7 +227,7 @@ public void onFailure(Object error) {
                 conversationBuilder.launchConversation(new KmCallback() {
                     @Override
                     public void onSuccess(Object message) {
-                        callback.invoke(SUCCESS, message != null ? message.toString() : "Success");
+                        callback.invoke(SUCCESS, message != null ? ChannelService.getInstance(currentActivity).getChannelByChannelKey((Integer) message).getClientGroupId()  : "Success");
```

Comment:
is this code internally used, or there is a possibility that some customer will calling this in existing code and comparing the return type with "success" string?